PSKoans/ExportedTypes.ps1

```diff
@@ -11,4 +25,33 @@ class KoanAttribute : System.Attribute {
     KoanAttribute() {
         $this.Position = [uint32]::MaxValue
     }
+
+    [void] InvokeHelpInfo() {
+        if ($this.HelpAction) {
+            $this.InvokeHelpAction()
+        }
+        elseif ($this.HelpPath) {
+            $this.InvokeHelpPath()
+        }
+        elseif ($this.HelpURL) {
+            $this.InvokeHelpURL()
+        }
+    }
+
+    [void] InvokeHelpAction() {
+        $this.HelpAction.InvokeReturnAsIs()
+    }
+
+    [void] InvokeHelpURL() {
+        Write-ConsoleLine "Confusion is only the beginning"
```

Comment:
This line is similar to some of the randomly-chosen params and is probably unneeded?